.evergreen.yml

```diff
@@ -135,14 +135,14 @@ functions:
       script: |
         export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
         yarn build-storybook
-
-  yarn-coverage:
+    
+  yarn-snapshot:
     command: shell.exec
     params:
       working_dir: spruce
       script: |
         export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
-        yarn coverage
+        yarn test --reporters="jest-junit" --reporters="default" --testPathPattern="storybook.test.ts"
```

Comment:
Does this need a `-u`? (honest question, I'm not sure what the answer is 😵‍💫)

Reply:
no it does not `-u` is a flag to regenerate snapshots. Which was causing the test to never fail since the failing snapshots would always be updated with updated ones.